cmd/ps-tag-onboarding/main.go

```diff
@@ -1,11 +1,11 @@
 package main
 
 import (
-	"github.com/murilogilfelpeto/ps-tag-onboarding-go/configuration"
-	"github.com/murilogilfelpeto/ps-tag-onboarding-go/handler"
-	"github.com/murilogilfelpeto/ps-tag-onboarding-go/repository"
-	"github.com/murilogilfelpeto/ps-tag-onboarding-go/router"
-	"github.com/murilogilfelpeto/ps-tag-onboarding-go/service"
+	"github.com/murilogilfelpeto/ps-tag-onboarding-go/internal/configuration"
+	"github.com/murilogilfelpeto/ps-tag-onboarding-go/internal/handler"
+	"github.com/murilogilfelpeto/ps-tag-onboarding-go/internal/repository"
+	"github.com/murilogilfelpeto/ps-tag-onboarding-go/internal/router"
+	"github.com/murilogilfelpeto/ps-tag-onboarding-go/internal/service"
 	log "github.com/sirupsen/logrus"
```

Comment:
IN all new development we are now using https://pkg.go.dev/log/slog for logging since it's built into go. For this exercise you don't need to switch to slog but I figure I would point it out for reference.

Reply:
I saw that in our projects, but for this project I thought that logrus fit better here.